libavcodec/vvc/sei.h

```diff
@@ -31,8 +31,19 @@
 #include "libavcodec/sei.h"
 #include "libavcodec/vvc.h"
 
+typedef struct VVCSEIPictureHash {
```

Comment:
could we define this in h274.h? and follow film grain things

Reply:
We can, but film grain is exposed to the API user with the AV prefix. It's used to align with hevc/sei.h

Reply:
but the vvc sei defined in h274, right?

Reply:
Yeah. Will do.